DQM/Integration/python/clients/beampixel_dqm_sourceclient-live_cfg.py

```diff
@@ -1,6 +1,7 @@
 import FWCore.ParameterSet.Config as cms
```

Comment:
are you sure this changes belong to your developments for trigger DQM/validation ?